### PR TITLE
lrec-2022-ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by mjpost** [comment]
Are they going to send the workshops, too?

**Comment by mjpost** [comment]
Can you add `joint.yaml`? That will make the review slightly easier.

**Comment by mjpost** [comment]
@xinru1414 I assume you'll remove the draft status, once this is ready? Also, did you see my comments above? Please mark them as resolved once you've had a chance to look at them.

**Comment by xinru1414** [comment]
@mjpost `joint.yaml` has been updated.

**Review by mjpost — APPROVED** [review]

File: data/yaml/name_variants.yaml [hunk 1]
```diff
@@ -5476,6 +5476,9 @@
 - canonical: {first: Yang, last: Liu}
   comment: Peking University
   id: yang-liu-pk
+- canonical: {first: Yang, last: Liu}
+  comment: Samsung Research Center Beijing
+  id: yang-liu-ss
 - canonical: {first: Yang, last: Liu}
```
[the remark this answers is not in the document]
Comment (mjpost) [reply data/yaml/name_variants.yaml:5482]: We're not, but too hard to tell, going to leave this.

File: data/yaml/sigs/siglex.yaml [hunk 2]
```diff
@@ -2,6 +2,8 @@ Name: Special Interest Group on the Lexicon (SIGLEX)
 ShortName: SIGLEX
 URL: http://www.siglex.org/
 Meetings:
+  - 2022:
+    - 2022.mwe-1 # Proceedings of The 18th Workshop on Multiword Expressions @LREC2022
   - 2022:
```
[the remark this answers is not in the document]
Comment (mjpost) [reply data/yaml/sigs/siglex.yaml:7]: Fixed.

**Pull request merged**